packages/deployment/app/cloudformation.template.yml

```diff
@@ -623,6 +638,14 @@ Resources:
           Value: "{{stackName}}"
       VpcId: {{vpc.vpcId}}
 
+  SecurityGroupSSHinbound:
```

Comment:
opening up the ssh port to the world is not a good idea. The security configuration should also move to the config.yml to make it easier for the users to change it.

Reply:
are you saying that we should instead add an ssh option in the config.xml to enable this?
That would be fine.

Reply:
yeah